src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1137,7 +1137,12 @@ class S6Importer final : public IParkImporter
                 auto dst2 = dst->AsTrack();
                 auto src2 = src->AsTrack();
 
-                dst2->SetTrackType(src2->GetTrackType());
+                auto rideType = _s6.rides[src2->GetRideIndex()].type;
+                track_type_t trackType = static_cast<track_type_t>(src2->GetTrackType());
+
+                if (trackType == TrackElemType::RotationControlToggleAlias && !TrackTypeIsBooster(rideType, trackType))
```

Comment:
`TrackTypeIsBooster` is now only used for RCT2 import/export. Please rename it RCT2TrackTypeIsBooster and move it to RCT2.cpp/h.